### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [ShwStone/shwblog#66] [Announcements]
# about/

自我介绍 juruo OIer，XJTU 少年班预科一， Linux 爱好者。 写文章完全取决于学到哪写到哪或者偶然的爱好。 如果以后写了什么项目会挂在这里的。 online-class-helper 网课de好帮手（给老师用的） HydroOJ Rejudge Script 为 HydroOJ 设计的重测脚本。可以重测指定题目和比赛的所有提交。

https://shwst.one/about/

**Comment by ShwStone** [comment]
20231227
从 Mist 变成了 Gemini
背景变成了珂朵莉
加了很多有趣的小东西

**Comment by WindskyMekdull** [comment]
好耶！
________________________________
发件人: ShwStone ***@***.***>
发送时间: 2023年12月27日 19:01
收件人: ShwStone/shwblog ***@***.***>
抄送: WindskyMekdull ***@***.***>; Comment ***@***.***>
主题: Re: [ShwStone/shwblog] about/ (Discussion #66)


20231227
从 Mist 变成了 Gemini
背景变成了珂朵莉
加了很多有趣的小东西

―

**Comment by REPT2K19** [comment]
狠狠学习

**Comment by YHFrozen** [comment]
该更新了介绍了唔——直升巨佬，icpc西部赛金牌，同学们的好科委，某人的好舍友唔

**Comment by Panda-Ghost** [comment]
反馈反馈，你这概率论的东西的分类怎么一会儿单在数学里，一会儿在数学的子项概率论里，一会儿在数学和外面的那个概率论里 (╬￣皿￣)
